english/poke-the-wall/eat-way-out/open-door/open-door.md

```diff
@@ -0,0 +1,3 @@
+You sit, half-drowned and coughing up melted chocolate, amazed that you made it out. Behind you, chocolate continues to ooze out the now open marshmallow door. As you gather your senses together you take a look at your surroundings. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.